lib/eventFactory.js

```diff
@@ -687,7 +694,7 @@ class EventFactory extends BaseFactory {
                                         modelConfig.creator = creator;
 
                                         return this.scm.decorateCommit({
-                                            scmUri: pipeline.scmUri,
+                                            scmUri: subscribedEvent ? subscribedScmConfig.scmUri : pipeline.scmUri,
```

Comment:
I think this change is not needed. Does the subscribed repo need to show pipeline repo's build ? Would it be confusing?